lib/qbwc/response/item_site_query_rs.rb

```diff
@@ -0,0 +1,86 @@
+module QBWC
+  module Response
+    class ItemSitesQueryRs
+      attr_reader :records
+
+      def initialize(records)
+        @records = records
+      end
+
+      def handle_error(errors, config)
+        errors.each do |error|
+          Persistence::Object.handle_error(config,
+                                           error.merge(context: 'Querying item site query'),
+                                           'inventories',
+                                           error[:request_id])
+        end
+      end
+
+      def process(config = {})
+        return if records.empty?
+        puts 'ItemSitesQueryRs#process'
+
+        receive_configs = config[:receive] || []
+        inventory_params = receive_configs.find { |c| c['inventories'] }
+
+
+        if inventory_params
+          payload = { inventories: inventories_to_flowlink }
+          config = { origin: 'quickbooks' }.merge config.reject{|k,v| k == :origin || k == "origin"}
+
+          poll_persistence = Persistence::Polling.new(config, payload)
+          poll_persistence.save_for_polling
+
+          inventory_params['inventories']['quickbooks_force_config'] = 'true'
+          params = inventory_params['inventories']
+
+          Persistence::Settings.new(params.with_indifferent_access).setup
+        end
+
+        nil
+      end
+
+      private
+
+      def inventories_to_flowlink
+        records.map do |record|
+          {
+            list_id: record['ListID'],
+            created_at: record['TimeCreated'],
+            updated_at: record['TimeModified'],
+            edit_sequence: record['EditSequence'],
+            full_name: full_name(record),
```

Comment:
I've been adding extra field for things like this so we have extra info in the transforms.
I'd suggest keeping `full_name`, but also adding:
`inventory_item_name: record['ItemInventoryRef']['FullName']` and `assembly_item_name: record['ItemInventoryAssemblyRef']['FullName']`

Also, I like adding `qbe_item_type: record['ItemInventoryRef'] ? 'qbe_inventory' : 'inventory_assembly'`

Reply:
Sounds good! I'll also take a second look in the documentation and see if there are other fields we could pull